dist/images/Dockerfile.fedora

```diff
@@ -25,11 +25,6 @@ RUN INSTALL_PKGS=" \
 	dnf install --best --refresh -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
 	dnf clean all && rm -rf /var/cache/dnf/*
 
-# ensure we pick up ovn-2.12.1-1.fc31
-# this should have no effect in the future once the
-# advisory repo is gone and can be removed
-RUN dnf install ovn --best  --enablerepo=updates-testing --advisory=FEDORA-2020-34b65e864b1 || true
-
```

Comment:
Not sure, but when i searched through the repo, i found two version 2.12.0-1 and 20.03.0-0, didn't found 2.12.1-1 as such, do i need to add any other repo to dnf to find this package ?

```
# dnf --showduplicates list ovn
Last metadata expiration check: 1:57:39 ago on Thu Mar  5 17:57:21 2020.
Installed Packages
ovn.x86_64                                            20.03.0-0.fc31                                            @updates
Available Packages
ovn.x86_64                                            2.12.0-1.fc31                                             fedora
ovn.x86_64                                            20.03.0-0.fc31                                            updates
```

Reply:
It's here:
https://bodhi.fedoraproject.org/updates/FEDORA-2020-34b65e864b

Yours may have just already been replaced by 20.03. You can see the latest builds here:
https://koji.fedoraproject.org/koji/packageinfo?packageID=28508